### PR TITLE
Extract two more helpers

### PR DESCRIPTION
**Pull request by AlekSi**
## Readiness checklist

* [ ] I added tests for new functionality or bugfixes.
* [x] I ran `task all`, and it passed.
* [x] I added/updated comments for both exported and unexported top-level declarations (functions, types, etc).
* [x] I checked comments rendering with `task godocs`.
* [x] (for maintainers only) I set Reviewers, Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: .golangci.yml [hunk 1]
```diff
@@ -114,6 +114,10 @@ linters:
 issues:
   exclude-use-default: false
   exclude-rules:
+    # ignore deprecation errors in existing code; new code is checked by the other configuration
```
Comment (rumyantseva) [inline .golangci.yml:117]: Awwww! Thanks!

**Review by rumyantseva — DISMISSED** [review]
LGTM

**Review by w84thesun — DISMISSED** [review]

**Review by noisersup — DISMISSED** [review]

**Comment by noisersup** [comment]
golangci-lint fails but if I understand correctly it won't fail after we merge it.

**Comment by AlekSi** [comment]
There was a problem with golangci-lint configuration. @noisersup @rumyantseva PTAL

File: internal/util/testutil/db.go [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+package testutil
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+// DatabaseName returns a stable database name for that test.
```
Comment (seeforschauer) [inline internal/util/testutil/db.go:24]: What does mean stable here?

**Review by seeforschauer — APPROVED** [review]

File: internal/util/testutil/db.go (hunk 2 above)
Comment (AlekSi) [reply internal/util/testutil/db.go:24]: The same for every invocation, non-random

**Review by rumyantseva — APPROVED** [review]
LGTM (the linter still doesn't pass though). Actually, now it looks like there is some error there...

**Comment by AlekSi** [comment]
The cache was reset after the Go version bump, and 2 minutes is not enough for golangci-lint with a clean cache. But normally, that's not a problem because cron pre-fills our caches at night.

**Pull request merged**